src/Data/toClientParams.ts

```diff
@@ -30,13 +47,55 @@ export function toClientParams(params: IndexParams): FetchParams {
   }
 }
 
-function toClientFilterParam(filters: Record<string, OperatorSpec>) {
+function toClientFilterParam(
```

Comment:
Here, you update the `toClientParams` functions to replicate the new logic that has been built into the SDK.

Question: Since the new SDK version supports dynamic headers and also supports object values, why do we even need this module (`toClientParams`) anymore? 

I would have expected the portal app to switch to using `restApi` everywhere and to remove this code.

Or is there still some feature missing in the SDK?

Reply:
It's only the binary processing that still causes the need for this code, right? 
Maybe that can be refactored somehow? 

Because I would really love to get rid of this module, since it creates a liability for every developer forking this repo.

Reply:
With https://github.com/Scrivito/scrivito-portal-app/pull/536 there are now only two remaining callers of `toClientFilterParam`/`toClientParams`: `Ticket` and `Message`, since both do need custom binary handling for during update/create.

To my knowledge the user of the SDK can either only use `restApi` or have to implement all `connection` functions. So if we want to support all of the filters etc. for `Ticket` and `Message` we would still need something like `toClientParams`.